index.html

```diff
@@ -83,27 +84,61 @@ <h1>
       var settings = {
         test: {
           stripeKey:   'pk_test_AQChgkHVYwyEZ7tQ6USRMMn700qcDhF3kr',
-          productSKU:  'sku_EzjGIEQvq3lPMM',
+          products: {
+            classic: { sku: 'sku_EzjGIEQvq3lPMM', price: 99 },
+            lewagon: {
+              sku:          'sku_FHZh5DJtsfNxhK',
+              price:        50,
+              priceMessage: 'One-shot payment of <strike>99€</strike> <strong>50€</strong>',
+              message:      'Enjoy our Le Wagon family discount ❤️'
+            }
+          },
           redirectURL: window.location.origin
         },
         live: {
           stripeKey:   'pk_live_LX30bNQfyM7kr7Mndz2AKIYv00RvtzsXWT',
-          productSKU:  'sku_EzjFh7OMJC0s4H',
+          products: {
+            classic: { sku: 'sku_EzjFh7OMJC0s4H', price: 99 },
+            lewagon: {
+              sku:          'sku_FHZSVwZfvntbam',
+              price:        50,
+              priceMessage: 'One-shot payment of <strike>99€</strike> <strong>50€</strong>',
+              message:      'Enjoy our Le Wagon family discount ❤️'
+            }
+          },
           redirectURL: 'https://www.solidrubydesign.com'
         }
       }
 
-      var mode           = window.location.hostname == 'www.solidrubydesign.com' ? 'live' : 'test';
+      const mode          = window.location.hostname == 'www.solidrubydesign.com' ? 'live' : 'test';
+      const mode_settings = settings[mode]
```

Comment:
```suggestion
      const modeSettings = settings[mode];
```

Then variable must be renamed everywhere.